integrations/jetty/documentation.yaml

```diff
@@ -0,0 +1,238 @@
+          context: "$1"
+      - pattern: "org.eclipse.jetty.server.handler<type=(\\w+), id=(\\d+)><>(\\w+)"
+        name: "jetty_server_handler_$3"
+        labels:
+          id: "$2"
+          type: "$1"
+      - pattern: "org.eclipse.jetty.server.session<type=(.*), id=(\\d+)><>(\\w+)"
+        name: "jetty_server_session_$3"
+        labels:
+          id: "$2"
+          type: "$1"
+      - pattern: "org.eclipse.jetty.server.session<context=(.*), type=(.*), id=(\\d+)><>(\\w+)"
+        name: "jetty_server_session_$4"
+        labels:
+          id: "$3"
+          type: "$2"
+          context: "$1"
+      - pattern: "org.eclipse.jetty.util<type=(\\w+), id=(\\d+)><>(\\w+)"
+        name: "jetty_util_$3"
+        labels:
+          id: "$2"
+          type: "$1"
+      - pattern: "org.eclipse.jetty.util.component<type=(\\w+), id=(\\d+)><>(\\w+)"
+        name: "jetty_util_component_$3"
+        labels:
+          id: "$2"
+          type: "$1"
+      - pattern: "org.eclipse.jetty.servlet<context=(.*), type=(\\w+), id=(\\d+)><>(\\w+)"
+        name: "jetty_servlet_$4"
+        labels:
+          id: "$3"
+          type: "$2"
+          context: "$1"
+      - pattern: "org.eclipse.jetty.servlet<context=(.*), type=(\\w+), name=(\\w+), id=(\\d+)><>(\\w+)"
+        name: "jetty_servlet_$5"
+        labels:
+          id: "$4"
+          type: "$2"
+          name: "$3"
+          context: "$1"
+      - pattern: "org.eclipse.jetty.util.thread<type=(\\w+), id=(\\d+)><>(\\w+)"
+        name: "jetty_util_thread_$3"
+        labels:
+          id: "$2"
+          type: "$1"
+      - pattern: "org.eclipse.jetty.util.thread.strategy<context=(.*), type=(\\w+), id=(\\d+)><>(\\w+)"
+        name: "jetty_util_thread_strategy_$4"
+        labels:
+          id: "$3"
+          type: "$2"
+          context: "$1"
+      - pattern: "org.eclipse.jetty.webapp<context=(.*), type=(\\w+), id=(\\d+)><>(\\w+)"
+        name: "jetty_webapp_$4"
+        labels:
+          id: "$3"
+          type: "$2"
+          context: "$1"
+      - pattern: "org.eclipse.jetty.websocket.common<context=(.*), type=(\\w+), id=(\\d+)><>(\\w+)"
+        name: "jetty_websocket_jakarta_common_$4"
+        labels:
+          id: "$3"
+          type: "$2"
+          context: "$1"
+      - pattern: "org.eclipse.jetty.websocket.jakarta.common<context=(.*), type=(\\w+), id=(\\d+)><>(\\w+)"
+        name: "jetty_websocket_jakarta_common_$4"
+        labels:
+          id: "$3"
+          type: "$2"
+          context: "$1"
+      - pattern: "org.eclipse.jetty.(.*)"
+        name: jetty_metric_$1
+  ---
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: jetty
+  spec:
+    selector:
+      matchLabels:
+  +     app.kubernetes.io/name: jetty
+    replicas: 1
+    template:
+      metadata:
+        labels:
+  +       app.kubernetes.io/name: jetty
+      spec:
+        containers:
+        - name: jetty
+          image: jetty:11-jdk17-alpine
+  +        ports:
```

Comment:
There seems to be an extra space here, in case the `+` is removed from the Deployment spec, this part is indented correctly, but the part from line 160 gets incorrectly indented.

Reply:
I'm not 100% sure what it was, but the indentation issue should be resolved now. Thanks!